libs/domain/feature-loaders/src/snapshot/SnapshotProposalLoader.ts

```diff
@@ -0,0 +1,187 @@
+import { createGraphQLClient, GraphQLClient, ProgramError } from "@banklessdao/util-data";
+import { InitContext, InternalLoadContext, LoadContext, ScheduleMode } from "@shared/util-loaders";
+import { Data, Nested, NonEmptyProperty, OptionalNumberArrayOf, OptionalObjectRef, OptionalProperty, OptionalStringArrayOf, Property, RequiredArrayRef, RequiredStringArrayOf } from "@banklessdao/util-schema";
```

Comment:
There are unused imports here, can you remove them?

Reply:
Done!